packages/pg/test/unit/connection-parameters/environment-variable-tests.js

```diff
@@ -3,6 +3,11 @@ var helper = require('../test-helper')
 const Suite = require('../../suite')
 
 var assert = require('assert')
+const fs = require('fs')
+
+const tmp = require('tmp')
```

Comment:
Another undeclared dependency. Can the files just be part of the repo instead of written dynamically by test code?